EIPS/eip-5320.md

```diff
@@ -0,0 +1,156 @@
+  /// @param _tokenId The identifier of the token to revoke.
+  function revoke(uint256 _tokenId) external;
+
+  /// @dev call this to manually send pending taxes to the collector.
+  /// This function should be callable by any external party.
+  /// @param _tokenId The identifier of the token to pay taxes for.
+  function collect(uint256 _tokenId) external;
+  
+  /// @dev call this to change the perceived valuation of the token.
+  /// @param _tokenId The identifier of the token to change valuation of.
+  /// @param _valuation The new valuation of the token.
+  function changeValuation(uint256 _tokenId, uint256 _valuation) external;
+}
+```
+
+### Caveats
+
+The taxes have to be collected manually, either through `collect`, or by interacting with the Harberger tokens.
+
+The taxes are paid upfront, which is less capital efficient.
+
+The functions of the standard are not passing the buyer or owner as parameter, instead, they use `msg.sender`.
+
+The funds are per token, not per account.
+
+## Rationale
+
+### [EIP-20](./eip-20.md) compatibility
+
+EIP-20 compliant tokens are a widespread industry standard and using native tokens is limiting.  
+
+### Offer, instead of buying at current valuation
+
+Instead of doing this, `buy` could have simply accepted the valuation of the item as the buy price. This, however, has frontrunning concerns, as the current owner can frontrun and increase the valuation.
+
+### Taxes paid upfront
+
+Taxes could have been paid in two different ways:
+
+- Owner gives an allowance to the Harberger contract, and occasionally call `collect` manually to route the payment through Harberger to the collector.
+- Owner pays taxes upfront, and will be reimbursed the excess when the item is bought, revoked, or manually defunded.
+
+The first option has certain edge cases if no parties collect taxes frequently. The owner could remove the allowance or lose their balance, and become unable to pay. This provides a window for attacks, particularly, griefing attacks if the contract does not charge extra fees for some interactions. Example:
+
+1. Buy token for almost nothing and value it very high.
+2. When someone calls collect, default on the taxes so that it's revoked.
+3. Repeat 1.
+
+### No view for the tax rate
+
+Tax rates could be more complex than a simple flat rate, for example, they could be progressive, or be lower or higher if the address meets some specific conditions. The taxes could have discounts if they are collected more frequently. The possibilities are too high to justify making the tax rate a standard view, globally or per item.
+
+### No commit reveal for buying tokens
+
+Commit reveal would add too much complexity to an ERC. It could be added as an extension, but it should not be part of this standard, since it already has too many functions.
+
+### Taxes and funds are per token, not per account
+
+This made for a simpler implementation, but it may be too naive. Please give feedback on how to implement this.
```

Comment:
I still need to give my thoughts (at a retreat this week, so might have to wait).